app/views/idv/in_person/state_id.html.erb

```diff
@@ -96,7 +96,6 @@
       %>
   </div>
 
-  <% if capture_secondary_id_enabled %>
     <div class="margin-bottom-4">
       <%= render ValidatedFieldComponent.new(
             name: :state_id_jurisdiction,
```

Comment:
My expectation is that we'd unindent all these blocks that were inside the conditional statements before? It does create some diff noise, but we can turn on "hide whitespace changes" to hide those (same for the rest of the file too)